website/docs/_intro.md

```diff
@@ -0,0 +1,87 @@
+# Skyhook
+
+[![Build](https://github.com/aerospike/skyhook/actions/workflows/build.yml/badge.svg)](https://github.com/aerospike/skyhook/actions/workflows/build.yml)
+
+Skyhook is a Redis API-compatible gateway to the [Aerospike](https://www.aerospike.com/) Database. Use Skyhook to quickly get your Redis client applications up and running on an Aerospike cluster.
+
+## Overview
+
+Skyhook is designed as a standalone server application written in Kotlin, which
+accepts Redis protocol commands and projects them to an Aerospike cluster using
+the Aerospike Java client under the hood. It uses [Netty](https://netty.io/) as
+a non-blocking I/O client-server framework.
+
+This project is now in **beta**. If you're an enterprise customer feel free to
+reach out to our support with feedback and feature requests.
+We appreciate feedback from the Aerospike community on
+[issues](https://github.com/aerospike/skyhook/issues)
+related to Skyhook.
+
+## Installation
+
+### Prerequisites
+
+- Java 8 or later
+- Aerospike Server version 4.9+
+
+### Installing
+
+Skyhook is distributed as a jar file which may be downloaded from https://github.com/aerospike/skyhook/releases/latest.
+
+### Running
+
+Usage:
+
+```text
+% java -jar skyhook-[version]-all.jar -h
+
+Usage: skyhook [-h] [-f=<configFile>]
+Redis to Aerospike proxy server
+  -f, --config-file=<configFile>
+               yaml formatted configuration file
+  -h, --help   display this help and exit
+```
+
+To run the server:
+
+```sh
+java -jar skyhook-[version]-all.jar -f config/server.yml
+```
+
+The configuration file carries all the settings the server needs and is in YAML
+format. An example configuration file can be found in the [`config`](https://github.com/aerospike/skyhook/blob/a0199da72222984c8417ccaa6e4a02064ed7224b/config/server.yml) folder of this repository.
+If no configuration file is specified, the default settings will be applied.
+
+```text
+[main] INFO  c.a.skyhook.SkyhookServer$Companion - Starting the Server...
+```
+
+Now the server is listening to the `config.redisPort` (default: 6379) and is ready to serve.
+
+If you wish to deploy Skyhook as a cluster of nodes, you can find some example configurations [here](scaling-out).
```

Comment:
Broken link here.

Reply:
thanks looks like from readme to website absolute link is needed. This is fixed.